en/gps_compass/rtk_gps.md

```diff
@@ -140,6 +157,70 @@ See the [EKF2 GPS Configuration](../advanced_config/tuning_the_ecl_ekf.md#gps) s
+The two GPS devices in this scenario are referred to as the *Moving Base* and *Rover*.
+
+### Supported Devices
+
+This is the subset of devices above that have a U-blox F9P:
+* [Freefly RTK GPS](../gps_compass/rtk_gps_freefly.md) (F9P)
+* [Holybro H-RTK F9P GNSS](../gps_compass/rtk_gps_holybro_h-rtk-f9p.md)
+* [SparkFun GPS-RTK2 Board - ZED-F9P](https://www.sparkfun.com/products/15136) (www.sparkfun.com)
+* [SIRIUS RTK GNSS ROVER (F9P)](https://store-drotek.com/911-1010-sirius-rtk-gnss-rover-f9p.html#/158-sensor-no_magnetometer) (store-drotek.com)
+
+
+### Configuration/Setup
+
+:::tip
+The general setup is described  in: [ZED-F9P Moving base applications (Application note)](https://www.u-blox.com/sites/default/files/ZED-F9P-MovingBase_AppNote_%28UBX-19009093%29.pdf).
+:::
+
+In overview:
+- The UART2 of the GPS devices need to be connected together (TXD2 of the "Moving Base" to RXD2 of the "Rover")
+- Connect UART1 on each of the GPS to (separate) unused UART's on the autopilot, and configure both of them as GPS with baudrate set to `Auto`.
+  The mapping is as follows:
+  - Main GPS = Rover
+  - Secondary GPS = Moving Base
+- Set [GPS_UBX_MODE](../advanced_config/parameter_reference.md#GPS_UBX_MODE) to `Heading` (1)
+- Reboot and wait until both devices have GPS reception.
+  `gps status` should then show the Main GPS going into RTK mode, which means the heading angle is available.
+
+:::note
+If using RTK with a fixed base station the secondary GPS will show the RTK state w.r.t. the base station.
+:::
+
+Ideally the two antennas should be identical, on the same level/horizontal plane and oriented the same way, and on an identical ground plane size and shape ([Application note](https://www.u-blox.com/sites/default/files/ZED-F9P-MovingBase_AppNote_%28UBX-19009093%29.pdf), section *System Level Considerations*)). 
+
+Otherwise the antennas can be positioned as needed, but the [GPS_YAW_OFFSET](../advanced_config/parameter_reference.md#GPS_YAW_OFFSET) must be configured.
+This offset is the angle made by the *baseline* (the line between the two GPS modules) relative to the vehicle x-axis (front/back axis, as shown [here](../config/flight_controller_orientation.md#calculating-orientation)).
+
+:::note
+The application note does not state the minimal required separation between modules.
+50cm has been used in test vehicles running PX4.
+:::
+
+To activate heading fusion for the attitude estimation, set the [EKF2_AID_MASK](../advanced_config/parameter_reference.md#EKF2_AID_MASK) parameter to enable *GPS yaw fusion*.
+
+
+## RTK GPS that Output Yaw 
+
+Some RTK GPS units (i.e. with multiple antennas) can output a yaw angle.
+This can be used instead of the heading from the magnetic compass, and as an alternative to [RTK Heading with Two FP9 modules](#rtk-gps-heading-setup-configuration-u-blox-f9p).
+
+###  Supported Devices
+
+* [Trimble MB-Two](../gps_compass/rtk_gps_trimble_mb_two.md)
+
+### Configuration
```

Comment:
This is also needed for the F9P setup, and GPS_YAW_OFFSET also applies to the Trimble.

Reply:
Thanks! I added GPS_YAW_OFFSET here, and also linked to the trimble2 docs because they have additional info. The FP9 setup already includes both GPS_YAW_OFFSET and EKF2_AID_MASK. I should perhaps separate out the config and hardware setup, but decided not to for now. 

Please confirm this ""